package.json

```diff
@@ -2,14 +2,14 @@
   "name": "minecraft-protocol",
   "version": "0.16.7-GH",
   "description": "Parse and serialize minecraft packets, plus authentication and encryption.",
-  "main": "index.js",
+  "main": "src/index.js",
   "repository": {
     "type": "git",
     "url": "git://github.com/PrismarineJS/node-minecraft-protocol.git"
   },
   "scripts": {
-    "prepublish": "gulp && require-self",
-    "test": "mocha --recursive --require intelli-espower-loader --require source-map-support/register --reporter spec"
+    "prepublish": "require-self",
+    "test": "mocha --harmony_destructuring --harmony_default_parameters --recursive --require intelli-espower-loader --require source-map-support/register --reporter spec"
```

Comment:
This is where I meant these aren't mocha options AFAIK. Except if mocha automatically passes all options to node ?